lineman/app/components/dashboard_page/dashboard_page_controller.coffee

```diff
@@ -2,39 +2,29 @@ angular.module('loomioApp').controller 'DashboardPageController', ($rootScope, R
   $rootScope.$broadcast('currentComponent', 'dashboardPage')
   $rootScope.$broadcast('setTitle', 'Dashboard')
 
-  @loaded =
-    sort_by_date:
-      show_all: 0
-      show_unread: 0
-      show_proposals: 0
-    sort_by_group:
-      show_all: 0
-      show_unread: 0
-      show_proposals: 0
+  @loaded = {}
   @perPage =
     sort_by_date: 25
     sort_by_group: 10
-  @groupThreadCounts =
-    hidden:    0
-    collapsed: 5
-    expanded:  10
 
   @sort   = -> CurrentUser.dashboardSort
   @filter = -> CurrentUser.dashboardFilter
 
+  @loadedCount = =>
+    @loaded[@sort()] = @loaded[@sort()] or {}
+    @loaded[@sort()][@filter()] = @loaded[@sort()][@filter()] or 0
+
+  @updateLoadedCount = =>
+    current = @loadedCount()
+    @loaded[@sort()][@filter()] = current + @perPage[@sort()]
```

Comment:
I haven't reintegrated the loadedCount() into the interface yet; it's primarily designed to limit us displaying a bunch of threads we've loaded in other places on the dashboard (which I much prefer to control; if you've been navigating around the app a bunch and land on the dashboard, I'd rather there be 25 threads there rather than 500, and just load more as you need em.)